RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,101 @@
+---
+rpip: #<to be assigned>
+title: Bounty Standards
+description: Describes standards and incentives for the production of effective bounty proposals.
```

Comment:
Is it the production of *all* bounty proposals or bounties by the GMC specifically that you're targeting? If just GMC, then consider the following amendment:

```suggestion
description: Describes standards and incentives for the production of effective bounty proposals by the GMC.
```

Reply:
Its all of them.